### ISSUE TITLE
eslint Reactjs garbage parsing error message

### ISSUE THREAD

**Issue by devautor** [yannickcr/eslint-plugin-react#1715]
**When defining an empty return block inside render function** of a React component, eslint is reporting this parsing error (in VS code, Windows). What does it imply, other than not using an empty return? 

```
export class TestMe extends Component {
      render() {
        return (
    
        );
      }
    }
```

Error:
```
Parsing error: Unexpected token
        [0m [90m 49 | [39m    [36mreturn[39m ([0m
        [0m [90m 50 | [39m[0m
        [0m[31m[1m>[22m[39m[90m 51 | [39m    )[33m;[39m[0m
        [0m [90m    | [39m    [31m[1m^[22m[39m[0m
        [0m [90m 52 | [39m  }[0m
        [0m [90m 53 | [39m}[0m
        [0m [90m 54 | [39m[0m
```

**Comment by ljharb** [comment]
@devautor it implies that `return ();` is invalid JS, so eslint is unable to parse it at all. It has nothing to do with this plugin; it's just invalid code.
